app/assets/v2/js/pages/process_bounty.js

```diff
@@ -10,6 +10,46 @@ window.onload = function(){
             $('input[name=terms]').attr('checked','checked');
         }
 
+        var estimateGas = function(issueURL, method, success_callback, failure_calllback, final_callback){
```

Comment:
Both of these functions were removed in the standard bounties/january 2018 PR @ethikz

Reply:
@mbeacom wouldn't it be safer to rebase it entirely with master rather than removing it here,
we might end up overlooking something else :/

> to err is to be human

Reply:
@thelostone-mc I don't disagree...  Honestly, at this point, I'm kind of leaning towards a cherrypick or patch against a fresh branch from `master`
@ethikz @owocki Thoughts? Don't mean to make more work for you, but the commit history alone might get a little weird.